app/models/keyword_engine.rb

```diff
@@ -0,0 +1,46 @@
+require 'matrix'
+require 'tf-idf-similarity'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.